FrontEnd/src/components/basics/ImageDropdown/ImageDropdown.module.scss

```diff
@@ -11,12 +11,16 @@
 
 .dropdown {
   flex-grow: 1;
+
   .dropdownMenu {
     max-height: 200px;
     overflow-y: auto;
     background-color: white;
     border-radius: $input-border-radius;
     border: 3px solid $border-color;
+    opacity: 1 !important;
+    top: auto !important;
+    pointer-events: auto !important;
```

Comment:
Is "auto" not the default value? Also, what about 'cursor', could you use that instead?

Reply:
this is to counter a known issue of standalone dropdown in Bootstrap: https://github.com/react-bootstrap/react-bootstrap/issues/5065

Reply:
Nitpick: I think you should add that explanation as a comment above the pointer-events line to make sure that nobody removes it in the future